.eslintrc.js

```diff
@@ -12,12 +12,14 @@ module.exports = {
     'jsx-a11y/label-has-for': 'off', // This is deprecated but in the recommended extension for some reason
     'jsx-a11y/media-has-caption': 'off',
     'jsx-a11y/no-onchange': 'off',
+    'no-alert': 0,
+    'no-console': 'warn',
+    'react-hooks/exhaustive-deps': 'error',
+    'react-hooks/rules-of-hooks': 'error',
+    'react/jsx-filename-extension': [1, { 'extensions': ['.tsx', '.jsx'] }],
```

Comment:
kind of prefer `'off'` `'warn'` and `'error'` rather than `0`, `1` & `2`

Reply:
Fair enough, I'll do another PR later on to fix this.